packages/cysync-core/src/pages/OnBoarding/Information.tsx

```diff
@@ -14,60 +16,103 @@ import {
-        <Typography variant="h5" color="heading" $textAlign="center">
-          <LangDisplay text={title} />
-        </Typography>
-        <Container $bgColor="list" direction="column">
-          <ListContainer
-            py={2}
-            pr={3}
-            pl={6}
-            direction="column"
-            width="full"
-            gap={16}
-          >
-            {listItems.map((listItem, index) => (
-              <ListItem key={`list-item-${index + 1}`} width="full">
-                <Typography
-                  variant="h6"
-                  color="muted"
-                  $textAlign="left"
-                  $letterSpacing={0.05}
-                >
-                  <LangDisplay text={listItem} />
-                </Typography>
-              </ListItem>
-            ))}
-          </ListContainer>
-        </Container>
-      </DialogBoxBody>
-      <DialogBoxFooter>
-        <Button
-          onClick={() => {
-            navigateTo(routes.onboarding.usage.path);
-          }}
-          variant="primary"
+  toNextPage: () => void;
+}> = ({ title, listItems, buttonText, toNextPage }) => (
+  <DialogBox direction="column" align="center" width={700}>
+    <DialogBoxBody direction="column" align="center" gap={48}>
+      <Typography variant="h5" color="heading" $textAlign="center">
+        <LangDisplay text={title} />
+      </Typography>
+      <Container $bgColor="list" direction="column">
+        <ListContainer
+          py={2}
+          pr={3}
+          pl={6}
+          direction="column"
+          width="full"
+          gap={16}
         >
-          <LangDisplay text={buttonText} />
-        </Button>
-      </DialogBoxFooter>
-    </DialogBox>
-  );
-};
+          {listItems.map((listItem, index) => (
+            <ListItem key={`list-item-${index + 1}`} width="full">
+              <Typography
+                variant="h6"
+                color="muted"
+                $textAlign="left"
+                $letterSpacing={0.05}
+              >
+                <LangDisplay text={listItem} />
+              </Typography>
+            </ListItem>
+          ))}
+        </ListContainer>
+      </Container>
+    </DialogBoxBody>
+    <DialogBoxFooter>
+      <Button
+        onClick={() => {
+          toNextPage();
+        }}
+        variant="primary"
+      >
+        <LangDisplay text={buttonText} />
+      </Button>
+    </DialogBoxFooter>
+  </DialogBox>
+);
 
 export const Information: React.FC = () => {
   const lang = useAppSelector(selectLanguage);
+  const location = useLocation();
+
+  const navigateTo = useNavigateTo();
+
+  const toNextPage = async () => {
+    // If the user has visited the page via back button, we don't skip the page.
+    if (location.search === routes.utilityRoutes.back.path) {
+      navigateTo(
+        routes.onboarding.usage.path + routes.utilityRoutes.forward.path,
+      );
+    } else {
+      navigateTo(routes.onboarding.usage.path);
+    }
+  };
```

Comment:
Use the `useQuery` hook to get the search parameter result. Don't compare the raw query parameter, since it'll not work if we have multiple queries.

Example:
- This is what we are comparing now
```ts
if ("?back=true" === "?back=true")
```

- Instead, we should do it like
```ts
if (query.get("back") === "true")
```

Check `CardAuthentication` Dialog box to see how `useQuery` is used.